packages/fxa-auth-server/config/dev.json

```diff
@@ -75,6 +76,7 @@
       "exampleProd3": [ "exampleCap1", "exampleCap2", "exampleCap3", "exampleCap4", "exampleCap5" ]
     },
     "clientCapabilities": {
+      "98e6508e88680e1a": [ "isRegistered", "isSubscribed" ],
```

Comment:
Should this be the FxA settings page client_id or the payments page? If the payment's page, please use a client_id of `59cceb6f8c32317c`

Reply:
Is this logic actually used anywhere or should this be part of a separate PR?

Reply:
I think is spillover from the demo I threw together this past Monday, I'll just remove.